standards/application/relay-reliability.md

```diff
@@ -0,0 +1,192 @@
+---
+title: TRANSPORT-RELIABILITY
+name: Waku Transport Reliability 
+category: Standards Track
+tags: [reliability, application]
+editor: Kaichao Sun <kaichao@status.im>
+contributors:
+  - Richard Ramos <richard@status.im>
+---
+
+## Abstract
+
+Waku is efficient for routing messages with its transport layer like [Relay protocol](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/11/relay.md) and [Lightpush](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/19/lightpush.md) / [Filter](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/12/filter.md) protocol. But there's no guarantee that a message going through Waku will reach to its destination. For example, the receiver in a chat application may miss messages when network issue happens at either sender or receiver side. 
```

Comment:
A bit nitpicky, but I would rephrase to make it clear that Waku Relay (with Filter/Lightpush) _is_ a p2p transport layer. If we phrase it like the transport layer just belongs to Waku, it may be confusing because the Waku Transport does indeed also build on hop-to-hop transport layers like QUIC (not implemented yet), TCP, websockets, etc.